src/main/scala/is/hail/annotations/ReadRowsRDD.scala

```diff
@@ -1,21 +1,49 @@
 package is.hail.annotations
```

Comment:
Meta-comment: I'm rather inclined not to pile several classes into one file whose name is one of those classes. Seems messy.